staging/src/k8s.io/pod-security-admission/metrics/metrics.go

```diff
@@ -16,9 +16,54 @@ limitations under the License.
 
 package metrics
 
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	admissionapi "k8s.io/pod-security-admission/admission/api"
+)
+
+const (
+	namespace = "pod_security"
+)
+
+var SecurityEvaluation = metrics.NewCounterVec(
+	&metrics.CounterOpts{
+		Namespace:      namespace,
+		Name:           "evaluations_total",
+		Help:           "Counter of pod security evaluations.",
+		StabilityLevel: metrics.ALPHA,
+	},
+	[]string{"decision", "policy_level", "policy_version", "mode", "operation", "resource"},
+)
+
 type EvaluationRecorder interface {
 	// TODO: fill in args required to record https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2579-psp-replacement#monitoring
-	RecordEvaluation()
+	RecordEvaluation(attrs admissionapi.Attributes, response *admissionapi.Response)
+}
+
+type PrometheusRecorder struct {
 }
 
-// TODO: default prometheus-based implementation
+func NewPrometheusRecorder() *PrometheusRecorder {
+	legacyregistry.MustRegister(SecurityEvaluation)
+	return &PrometheusRecorder{}
+}
+
+func (r PrometheusRecorder) RecordEvaluation(attrs admissionapi.Attributes, response *admissionapi.Response) {
```

Comment:
I think this method signature should be something like:
```suggestion
func (r PrometheusRecorder) RecordEvaluation(decision Decision, policy api.LevelVersion, evalMode Mode, attrs admissionapi.Attributes) {
```

Reply:
Actually, we may want to include an `policy.AggregatedCheckResult` argument as well. We will ignore it for our default prometheus implementation, but providing it might be useful to folks customizing the metrics implementation.

Reply:
Why we need the `evalMode`? I think `evalMode` can get from the policy?😳

Reply:
Why we need to use `levelVersion and evalMode`? I think it can get from `apiPolicy`😳